src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1422,3 +1422,53 @@ function random_orientation_dag(
     end
     return g2
 end
+
+"""
+    bernoulli_graph(Λ)
+
+Given the symmetric matrix ``\\Lambda \\in [0,1]^{n \\times n}``, return a Bernoulli graph with ``n`` vertices. Each edge ``(i,j)`` exists with probability ``\\operatorname{Bernoulli}(\\Lambda[i,j])``.
```

Comment:
```suggestion
Given the symmetric matrix ``\\Lambda \\in [0,1]^{n \\times n}``, return a Bernoulli graph with ``n`` vertices. Each edge ``(i,j)`` exists with probability ``\\Lambda[i,j]``.
```
should be enough, right?

Reply:
Also, while I am not completely against the use of latex symbols in the docstrings, one should also keep in mind that these docstrings are often read in the REPL in a terminal, so one should use them sparingly.